### PR TITLE
Housekeeping - Update Spelling and minor code fixes

### PR DESCRIPTION
**Pull request by ChrisPulman**
**What kind of change does this PR introduce?**

Update

**What is the current behavior?**

Code has spelling errors

**What is the new behavior?**

Fixed ReactiveUI main project spelling

**What might this PR break?**

none expected

**Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Other information**:

### CODE REVIEW

File: src/ReactiveUI/Bindings/Property/IPropertyBinderImplementation.cs [hunk 1]
```diff
@@ -82,7 +82,7 @@ public interface IPropertyBinderImplementation : IEnableLogger
     /// This can be a child property, for example <c>x =&gt; x.Foo.Bar.Baz</c> in which case
     /// the binding will attempt to subscribe recursively to updates in order to
     /// always get and set the correct property.</param>
-    /// <param name="signalViewUpdate">An observable, that when signaled, indicates that the view property
+    /// <param name="signalViewUpdate">An observable, that when signalled, indicates that the view property
```
Comment (glennawatson) [inline src/ReactiveUI/Bindings/Property/IPropertyBinderImplementation.cs:85]: Signaled is US english which is generally where we have the code base btw

File: src/ReactiveUI/Interactions/Interaction.cs [hunk 2]
```diff
@@ -24,7 +24,7 @@ namespace ReactiveUI;
 /// By default, handlers are invoked in reverse order of registration. That is, handlers registered later
 /// are given the opportunity to handle interactions before handlers that were registered earlier. This
 /// chaining mechanism enables handlers to be registered temporarily in a specific context, such that
-/// interactions can be handled differently according to the situation. This behavior can be modified
+/// interactions can be handled differently according to the situation. This behaviour can be modified
```
Comment (glennawatson) [inline src/ReactiveUI/Interactions/Interaction.cs:27]: Generally we been using US english as a standard

File: src/ReactiveUI/Bindings/Property/IPropertyBinderImplementation.cs (hunk 1 above)
Comment (ChrisPulman) [reply src/ReactiveUI/Bindings/Property/IPropertyBinderImplementation.cs:85]: I used an US English dictionary. Strange that it picked that up as incorrect.

Comment (glennawatson) [reply src/ReactiveUI/Bindings/Property/IPropertyBinderImplementation.cs:85]: Yeah it seems to be correcting to UK english elsewhere with behaviour vs behavior etc

File: src/ReactiveUI/Interactions/Interaction.cs (hunk 2 above)
Comment (ChrisPulman) [reply src/ReactiveUI/Interactions/Interaction.cs]: All should be reverted now, it looks like the US dictionary built into VS2022 has issues :) it has a British accent.

File: src/ReactiveUI/Bindings/Property/IPropertyBinderImplementation.cs (hunk 1 above)
Comment (ChrisPulman) [reply src/ReactiveUI/Bindings/Property/IPropertyBinderImplementation.cs]: Updated

**Review by glennawatson — APPROVED** [review]

**Pull request merged**